sql3/planner/compilealtertable.go

```diff
@@ -92,7 +92,7 @@ func (p *ExecutionPlanner) analyzeAlterTableStatement(stmt *parser.AlterTableSta
 			return sql3.NewErrUnknownType(col.Type.Name.NamePos.Line, col.Type.Name.NamePos.Column, typeName)
 		}
 
-		if strings.ToLower(columnName) == "_id" {
+		if strings.ToLower(columnName) == string(dax.PrimaryKeyFieldName) {
```

Comment:
Do all tables have PrimaryKeyFieldName?  How is this field denoted?

Reply:
All on-disk FeatureBase tables have an "_id" column implied - this just replaces the "_id" string with a constant.